src/olympia/devhub/templates/devhub/addons/submit/describe.html

```diff
@@ -93,12 +94,14 @@
     </div>
     {% if license_form %}
       <b>{{ _('License:') }}</b>
+      {{ license_form.builtin.errors }}
+      {% if addon.type == amo.ADDON_STATICTHEME %}
+        {% include 'devhub/addons/includes/cc_license_chooser.html' %}
+      <div class="addon-submission-field" id="license-list">
+      {% else %}
       <div class="addon-submission-field">
```

Comment:
I don't have a good solution, but man, the indentation is a mess in this file...

Reply:
its fixable, just with a lot more diff noise.

Reply:
And there's always the trade-off between indentation with `{% if ... %}` in the template meaning the resulting html is oddly indented.

Reply:
Yeah it's going to look bad regardless, don't worry about it too much

Reply:
didn't end up being that bad https://github.com/mozilla/addons-server/pull/6740/commits/b8c370c2c5ca9c38f5c6ecf876922a149152f200